packages/sit-onyx/src/components/examples/FormExample/FormExample.vue

```diff
@@ -59,7 +59,7 @@ const fruitOptions = ref(
     "Melon",
     "Raspberry",
     "Strawberry",
-  ].map<SelectOption>((option) => ({ value: option.toLowerCase(), label: option })),
+  ].map<SelectOption<string>>((option) => ({ value: option.toLowerCase(), label: option })),
```

Comment:
We could also use `satisfies SelectOption` so the type is inferred automatically